### PR TITLE
soxi: add page

### PR DESCRIPTION
**Pull request by Njoselson**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/soxi.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# soxi
+
+> SoXI - Sound eXchange Information, display sound file metadata.
+> More information: <https://linux.die.net/man/1/soxi>.
+
+- Display sound file metadata:
+
+`soxi path/to/file.wav`
```
Comment (kbdharun) [inline pages/common/soxi.md:8]:
```suggestion
> More information: <https://manned.org/soxi.1>.

- Display the sound file metadata:

`soxi {{path/to/file.wav}}`
```

Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Note: We prefer manned links when the author's documentation link isn't present.

**Comment by Njoselson** [comment]
Agreed! Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**